lib/domain/snapshot.js

```diff
@@ -24,4 +24,7 @@ export default class Snapshot {
     this.notes = notes || '';
     this.vulnerabilities = vulnerabilities || [];
   }
+
+  // dynamodb ttl
+  ttl;
```

Comment:
Not sure this is needed? We can set `ttl` on the object without defining it upfront. I guess this is useful for documenting that the property exists though?

Reply:
yeah that was really the only reasoning behind it